### PR TITLE
Refer the user to the build directory

### PR DESCRIPTION
**Pull request by ganwell**
If the user attempted an in-source build, use ./build as the build
directory and print a message referring the user to that directory.

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -566,9 +565,10 @@ for line in sys.stdin.readlines():
 
 
 if __name__ == "__main__":
-    if os.path.dirname(__file__) == os.curdir:
-        print("Please configure in empty build dir.")
-        os.exit(1)
+    in_source_attempt = os.path.dirname(__file__) == os.curdir
+    if in_source_attempt:
+        os.chdir("build")
```
Comment (winged) [inline configure]: Can we safely assume that the build folder exists? Or should we add a `os.mkdir` call in here to be sure?

**Comment by ganwell** [comment]
@winged Creating the directory. Will probably never happen, but it doesn't hurt.

File: configure [hunk 2]
```diff
@@ -577,3 +581,5 @@ if __name__ == "__main__":
     write_ide_files(args)
     write_pfix(rg_list)
     test_depends(args)
+    if in_source_attempt:
+        print("\nTo build chirp call:\n\ncd build\nmake all")
```
Comment (winged) [inline configure]: I would print this in any case, doesn't hurt to tell the user that we infact handled the build dir parameter correctly

Comment (ganwell) [reply configure]: But the message is wrong, if we print it in the other case.

Comment (winged) [reply configure]: Should of course be "...cd %s..."  % _build :)

Comment (ganwell) [reply configure]: @winged ok. implemented a dynamic message.

**Pull request merged**